jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -78,17 +78,16 @@ class CodeModernizerSession(
             }
             val startTime = Instant.now()
             val result = sessionContext.createZipWithModuleFiles()
-            payload = when (result) {
-                is ZipCreationResult.Missing1P -> {
-                    notifyStickyInfo(
-                        message("codemodernizer.notification.info.maven_failed.title"),
-                        message("codemodernizer.notification.info.maven_failed.content")
-                    )
-                    result.payload
-                }
 
-                is ZipCreationResult.Succeeded -> result.payload
+            if(result is ZipCreationResult.Missing1P){
```

Comment:
New behavior: If maven fails we do NOT want the users to move forward.

To help increase efficacy of acceptance rates, we need to make sure there is more validation on the front end.